### ISSUE TITLE
Minor bug in Script Window message

### ISSUE THREAD

**Issue by lilyclements** [africanmathsinitiative/R-Instat#7931]
@Patowhiz can you assign this to who can fix it? It is a very minor one that I came across! But in ucrScript, we currently have

https://github.com/africanmathsinitiative/R-Instat/blob/ab00c9ea6ecfbb9c99bdd94812bc674307130212/instat/ucrScript.vb#L383

```
    Private Sub mnuPaste_Click(sender As Object, e As EventArgs) Handles mnuPaste.Click
        If Clipboard.ContainsData(DataFormats.Text) Then
            txtScript.Paste()
            EnableDisableButtons()
        Else
            MsgBox("You can only paste text data on the script window.", "Paste to Script Window")
        End If
    End Sub
```

If I try and paste something in the script window and I get the error message, it crashes. I think the issue is that we do not want to use `MsgBox` here, but instead have a different function. I'm not sure which function though. Happy to fix it if you can tell me the function, otherwise, do you know who can?

**Comment by Patowhiz** [comment]
@lilyclements you mean even text pasting into the script window?
Are you using the development version of R-Instat or the release version?
@lloyddewit recently added a new control to the script window. I didn't get any bug when I pasted to it.

**Comment by lilyclements** [comment]
@Patowhiz pasting text is fine, because we shouldn't get the error message then. The issue I'm getting is with the error message. We should have a message pop up if I try to paste something that I shouldn't - but instead the program crashes because our error message is incorrect.

**Comment by Patowhiz** [comment]
@lilyclements it's not crashing on my end, neither I'm I getting the message. How can I reproduce the error message. I'm using the latest changes from the master.

**Comment by lilyclements** [comment]
@Patowhiz I am using the latest version too - it occurs for me if I paste something like a picture not from the R environment. Like if I paste a photo from my PC (I only discovered the issue because I thought I had copied something else)

**Comment by Patowhiz** [comment]
@lilyclements okay. @anastasia-mbithe kindly try and reproduce this too, as outlined above by @lilyclements. Use the latest changes from the master. It would be useful if you investigate at what point it breaks. Many thanks.

**Comment by anastasia-mbithe** [comment]
@Patowhiz and @lilyclements, am not able to reproduce the error as well, if I paste an image/picture, nothing happens; no error message, no crashing.

**Comment by N-thony** [comment]
> @Patowhiz and @lilyclements, am not able to reproduce the error as well, if I paste an image/picture, nothing happens; no error message, no crashing.

Same to me, I was not able to reproduce this error.

**Comment by lilyclements** [comment]
The issue isn't about triggering the error message - the issue is that when the error message is triggered, R-Instat crashes.
We want the error message to be triggered, but of course don't want to crash R-Instat.
So we need to find out how to trigger the error message for you so that we can work out if it causes a crash.

I have worked out the problem that was causing the crash, and so I have fixed it here #7935
